source/minishift/index.html.erb

```diff
@@ -0,0 +1,123 @@
+---
+title: Minishift - Containerized OpenShift Cluster
+---
+
+<% content_for :header do %>
+<h1 class="animated fadeInDown delay">
+  Minishift
+</h1>
+<p class="animated fadeInUp delay">Develop Applications in a Local Containerized OpenShift Cluster</p>
```

Comment:
Personally, I find "Local Containerized OpenShift Cluster" a bit misleading. What does "Local" really mean? IMO something like "Develop Applications in a Containerized OpenShift Cluster on your local host". I think this is much less ambiguous.